docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -28,19 +28,37 @@ Combobox is composed of three main parts:
 #### Trigger
 
 The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
-
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+initiate option selection. When implementing Combobox, you may rely on the
+default trigger or, you may pass in a custom activator.
+
+The default trigger is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing a
+label that reads "Select" and an `add` Icon suffix. The label can be customized
+by adding the optional `label` prop to better reflect the type of data being
```

Comment:
I mentioned it above but either via sub heading in this section, or adding some more context let's emphasize that the optional `label` prop to `Combobox` only affects the default trigger

Reply:
Ok - I created subheadings here: a027f48c8f8ec63183f53a816e2d9d30e70e126d
I made some of the verbiage around `label` more clear